### PR TITLE
Fixed production paths for themes files

### PR DESCRIPTION
**Pull request by josaphatim**
Generation of production files was making Cypht to miss themes files. As solution I copied content from vendor/thomaspark/bootswatch to respective assets in module.
And also fixed 2fa setting css by the way

### CODE REVIEW

File: scripts/config_gen.php [hunk 1]
```diff
@@ -235,8 +237,7 @@ function combine_includes($js, $js_compress, $css, $css_compress, $settings) {
     $js_hash = '';
     $css_hash = '';
     if ($css) {
-        $css_out = file_get_contents(VENDOR_PATH . "twbs/bootstrap/dist/css/bootstrap.min.css");
-        $css_out .= file_get_contents(VENDOR_PATH . "twbs/bootstrap-icons/font/bootstrap-icons.css");
+        $css_out = file_get_contents(VENDOR_PATH . "twbs/bootstrap-icons/font/bootstrap-icons.css");
```
Comment (kroky) [inline scripts/config_gen.php:240]: Please make sure this file is included after the bootstrap main css file.

Comment (kroky) [reply scripts/config_gen.php:240]: Actually, why don't we add the bootstrap-icons as well to the bootswatch pipeline?

Comment (kroky) [reply scripts/config_gen.php:240]: Great, thanks!

File: modules/themes/modules.php [hunk 2]
```diff
@@ -65,7 +65,7 @@ protected function output()
     {
         if ($this->get('theme') && in_array($this->get('theme'), array_keys($this->get('themes', array())), true)) {
             $theme_name = $this->html_safe($this->get('theme'));
-            return '<link href="' . WEB_ROOT . 'modules/themes/assets/' . $theme_name . '/css/' . $theme_name . '.css?v=' . CACHE_ID . '" media="all" rel="stylesheet" type="text/css" />';
+            return '<link href="modules/themes/assets/' . $theme_name . '/css/' . $theme_name . '.css?v=' . CACHE_ID . '" media="all" rel="stylesheet" type="text/css" />';
```
[the remark this answers is not in the document]
Comment (josaphatim) [reply modules/themes/modules.php]: I have fixed this too.

**Pull request merged**